addon/styles/_frost-link.scss

```diff
@@ -121,6 +121,7 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
   &.tertiary {
     background-color: inherit;
     color: $frost-color-blue-1;
+    margin-bottom: -7px;
```

Comment:
You've cross-browser tested the margin-bottom with different font sizes then?  Sorry, I'm not trying to be problematic here, just need to know if this is a more significant issue